plugin/open_url.vim

```diff
@@ -3,13 +3,13 @@ if exists('g:loaded_open_url')
 endif
 let g:loaded_open_url = 1
 
-function! s:SetGlobalOpt(opt, val)
-  if !exists('g:'.a:opt)
-    let g:{a:opt} = a:val
-  endif
-endfunction
+if !exists('g:open_url_browser_default')
+  let g:open_url_browser_default = 'xdg-open'
+endif
 
-call s:SetGlobalOpt('open_url_browser_default', 'sensible-browser')
+if !exists('g:open_url_default_mappings')
+  let g:open_url_default_mappings = 1
+endif
```

Comment:
Utilize the helper `call s:SetGlobalOpt('open_url_default_mappings', 1)` for better readability.

Reply:
Done